app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    file_name, new_path = command.split(" ")[1:]
+
+    with open(file_name, "r") as sorce:
```

Comment:
There is a typo in the variable name `sorce`. It should be `source` to correctly represent the file being read.